src/test/java/nextstep/subway/domain/line/LineTest.java

```diff
@@ -0,0 +1,111 @@
+package nextstep.subway.domain.line;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Section;
+import nextstep.subway.domain.Station;
+import nextstep.subway.exception.LineException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("Line 클래스")
+public class LineTest {
+
+    @Nested
+    @DisplayName("생성자는")
+    class Describe_constructor {
+
+        @DisplayName("이름이 null이면 예외를 반환")
+        @Test
+        void context_with_null_name_returns_exception() {
+            assertThatThrownBy(() -> new Line(null, "green"))
+                    .isInstanceOf(LineException.class)
+                    .hasMessage("노선이름을 입력해야합니다");
+        }
+
+        @DisplayName("이름이 빈값이면 예외를 반환")
+        @Test
+        void context_with_empty_name_returns_exception() {
+            assertThatThrownBy(() -> new Line("", "green"))
+                    .isInstanceOf(LineException.class)
+                    .hasMessage("노선이름을 입력해야합니다");
+        }
+
+        @DisplayName("color가 null이면 예외를 반환")
+        @Test
+        void context_with_null_color_returns_exception() {
+            assertThatThrownBy(() -> new Line("2호선", null))
+                    .isInstanceOf(LineException.class)
+                    .hasMessage("노선컬러를 입력해야합니다");
+        }
+
+        @DisplayName("color가 빈값이면 예외를 반환")
+        @Test
+        void context_with_empty_color_returns_exception() {
+            assertThatThrownBy(() -> new Line("2호선", ""))
+                    .isInstanceOf(LineException.class)
+                    .hasMessage("노선컬러를 입력해야합니다");
+        }
+
+    }
+
+    @Nested
+    @DisplayName("addSection 메소드는")
+    class Describe_addSection {
```

Comment:
```suggestion
    class DescribeAddSection {
```
클래스명이므로, 카멜로 붙여서 써도 좋을 것 같습니다..! 그런데 혹시 inner class 로 만들어주신 이유가 있을까요..?